tests/functional/testSubscriptionNeeded.js

```diff
@@ -0,0 +1,172 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ const assert = require('assert');
+ const vpn = require('./helper.js');
+ const queries = require('./queries.js');
+ const { startAndConnect } = require('./setupVpn.js');
+
+describe("subscription needed tests", function() {
+  // TODO: Create tests for this screen.
```

Comment:
Is there a follow up issue for this TODO? Or are you intending on adding additional tests later in this PR?

Reply:
Good call. I'll file a ticket.